applications/ConstitutiveLawsApplication/custom_constitutive/composites/rule_of_mixtures_law.h

```diff
@@ -649,6 +649,17 @@ class KRATOS_API(CONSTITUTIVE_LAWS_APPLICATION) ParallelRuleOfMixturesLaw
 
 private:
 
+    /**
+     * @brief This method returns true if the variable is related to a layer
+     * @param rVariable The variable to be printed
+     */
+    template<class TVariableType>
+    const bool IsLayerVariable(const TVariableType& rVariable)
+    {
+        // returns true if LAYER is found
+        return (rVariable.Name().find("LAYER") != std::string::npos);
```

Comment:
String find is sloooow

Reply:
I could basically repeat a lot of if...else if with all the new variables but I though that in this way is easier to maintain

Reply:
Yes, I am thinking something else, like for example checking the variable Id if all variables of layer have a way to be filtered. Thinking. Luckily this is only post-process, but I need to think about it.